<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMain</class>
 <widget class="QDialog" name="DlgMain">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SOSI file converter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grpInput">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lblSosiFiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>SOSI files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lstSosiFiles">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="grpOutput">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>300</height>
            </size>
           </property>
           <property name="title">
            <string>Output</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>12</number>
            </property>
            <item>
             <widget class="QLabel" name="lblFileFormat">
              <property name="text">
               <string>File format</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="tabConversion">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabShp">
               <property name="accessibleName">
                <string/>
               </property>
               <attribute name="title">
                <string>Shapefile</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </widget>
              <widget class="QWidget" name="tabSql">
               <attribute name="title">
                <string>PostGIS</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="txtCommandLine">
           <property name="minimumSize">
            <size>
             <width>340</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>340</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRunSosicon">
           <property name="minimumSize">
            <size>
             <width>340</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Run Sosicon</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="lstConsoleOutput"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnRunSosicon</sender>
   <signal>clicked()</signal>
   <receiver>DlgMain</receiver>
   <slot>onRunSosicon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>DlgMain</receiver>
   <slot>onAddSosiFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabConversion</sender>
   <signal>currentChanged(int)</signal>
   <receiver>DlgMain</receiver>
   <slot>onConversionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
